Sources/Extensions/UIViewController+Hero.swift

```diff
@@ -106,35 +69,87 @@ public extension UIViewController {
   }
 }
 
+//swift only
+extension UIViewController {
```

Comment:
I'm sorry for having to split out these extensions like so, but objective c doesn't support these types of enums